rfc6979.go

```diff
@@ -33,12 +33,10 @@ func hashBytesRFC6979(data []byte) []byte {
 //
 // Note that FIPS 186-3 section 4.6 specifies that the hash should be truncated
 // to the byte-length of the subgroup. This function does not perform that.
+//
+// Error always nil.
 func SignRFC6979(key *ecdsa.PrivateKey, msg []byte) ([]byte, error) {
-	r, s, err := rfc6979.SignECDSA(key, hashBytesRFC6979(msg), sha256.New)
-	if err != nil {
-		return nil, err
-	}
-
+	r, s := rfc6979.SignECDSA(key, hashBytesRFC6979(msg), sha256.New)
```

Comment:
`key == nil` leads to panic there. Maybe return error as in `ecsa.go` ?
```go
func Verify(pub *ecdsa.PublicKey, msg, sig []byte) error {
	. . .
	} else if pub == nil {
		return ErrEmptyPublicKey
	. . .
}
```

Reply:
yeah, you're right! thx